server/src/server.ts

```diff
@@ -51,7 +52,7 @@ let currentActiveTextDocument: TextDocument = TextDocument.create(
 )
 
 connection.onInitialize(async (params: InitializeParams): Promise<InitializeResult> => {
-  logger.level = 'debug'
+  logger.level = params.initializationOptions?.loggingLevel ?? 'error'

```

Comment:
the commit "Fix: Keep server settings in case a change is not defined " has already been merged. Try to rebase your branch on an up-to-date staging.

Reply:
Fixed.